.actions/assistant.py

```diff
@@ -68,12 +66,8 @@ def _replace_imports(lines: List[str], mapping: List[Tuple[str, str]]) -> List[s
 def copy_replace_imports(
     source_dir: str, source_imports: List[str], target_imports: List[str], target_dir: Optional[str] = None
 ) -> None:
-    """Copy package content with import adjustments.
-
-    >>> _ = copy_replace_imports(os.path.join(
```

Comment:
well, after simplification in #15365
it shall be included here: https://github.com/Lightning-AI/lightning/blob/f9d906c3405807da2167d6d2282e5b9cdbb1671e/.github/workflows/ci-pkg-install.yml#L46